app/src/main/java/org/schabi/newpipe/util/DeviceUtils.java

```diff
@@ -55,6 +57,16 @@ public static boolean isTv(final Context context) {
     }
 
     public static boolean isTablet(@NonNull final Context context) {
+        final String tabletModeKey = context.getString(R.string.tablet_mode_key);
+        final String tabletModeSetting = PreferenceManager.getDefaultSharedPreferences(context)
+                                                    .getString(tabletModeKey, "");
```

Comment:
I am not an Android developer or not very familiar with Android apps, so I cannot review the code itself but I think if the indentation isn't correct on these lines. Here is what I got when I paste the code in Android Studio:
```suggestion

        final String tabletModeSetting = PreferenceManager.getDefaultSharedPreferences(context)
                .getString(tabletModeKey, "");
```

Reply:
This kind of indentation is pretty much preference. There are coding styles that say to indent the one way or the other. From what I could find, the contribution guidelines only said to stick with checkstyle, which hasn't flagged this.

But either way, I don't really care. So if that's the preferred way, I can change it like that.